.ci-config/rippled.cfg

```diff
@@ -0,0 +1,184 @@
+[server]
```

Comment:
This `rippled.cfg` is for standalone nodes right ?

Reply:
Yes!